data/json/items/archery.json

```diff
@@ -276,10 +276,12 @@
       "symbol" : "(",
       "color" : "yellow",
       "name" : "self bow",
-      "description" : "A bow made from a single piece of wood, made specifically for the person using it.",
+      "description" : "A primitive bow made from a single piece of wood, made specifically for the person using it. Not that well, unfortunately...",
```

Comment:
You need to put two whitespaces between sentences.

Reply:
Damn. I thought I got all of those.